cypress.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  video: false,
+  defaultCommandTimeout: 10000,
+  e2e: {
+    specPattern: "cypress/e2e/**/*spec.ts",
+    baseUrl: "http://localhost:3000",
+  },
+  component: {
+    devServer: {
+      framework: "svelte",
```

Comment:
Nice! That you found this update! 🎉  💪

Reply:
Just for reference it seems that component testing now also works with node v17

Reply:
Are you referring to https://github.com/radicle-dev/radicle-interface/commit/d49130163d04eed45fbf48d5f572e38d11c06654?

Reply:
Didn't think about that commit, just for local development, but I think we could also remove that version restriction

Reply:
Bumped everything to node 18: 8fc669706162251111a984c3624f9902adf7ccef.